language/ir-testsuite/tests/signer/invalid_move_to_sender.mvir

```diff
@@ -0,0 +1,17 @@
+module M {
+    t(s: signer) {
+        move_to_sender<signer>(move(s));
+        return;
+    }
+}
+// check: ParserError: Invalid Token
```

Comment:
Interesting example! 

I guess a broader question is what the bytecode verifier should do with this code (i.e.,  should it accept or reject `move_to_sender<signer>`, `move_from<signer>`, `borrow_global<signer>`, etc). Is it possible to write these in bytecode (even if the IR parser won't allow them)? 

If so, it seems like the bytecode verifier ought to reject them (at least conceptually) because these instructions are supposed to work on a resource declared in the current module, and  `signer` is always declared elsewhere (by construction).

Reply:
The bytecode verifier can't reject them since it isn't expressible at the file format level. The instructions take a `StructDefinitionIndex`, so it will always be safe

Reply:
Ah, nice! I knew it used to be that way but didn't realize this is still true after the file format refactor. Excellent news.

Reply:
Nice! I didn't realize this was still true after the big file format refactor; excellent news.

Might be better to use storage op other than `move_to_sender` so we don't have to change this when we eliminate `move_to_sender`.

Reply:
I'm going to keep it at move_to_sender (even though it will need to be refactored) as the test is a bit closer to what I want (trying to actually move a `signer` into global storage directly)